package.json

```diff
@@ -6,6 +6,7 @@
     "@octokit/rest": "16.26.0",
     "@wordpress/env": "^4.2.2",
     "@wordpress/scripts": "^19.0",
+    "@wordpress/browserslist-config": "^4.1.2",
```

Comment:
This is included in the `@wordpress/scripts` package, so no need to add it separately

https://github.com/WordPress/gutenberg/blob/83a5df739dc904bc106a36002674aa7c68130aee/packages/scripts/package.json#L39

Reply:
yep! Thank you, good catch! fixed!